controllers/authController.js

```diff
@@ -0,0 +1,66 @@
+const User = require("../models/User");
+
+// Age of cookie (7 days)
+const maxAge = 7 * 24 * 60 * 60;
+
+// Handle errors
+const handleErrors = (err) => {
+	let errors = {
+		firstName: "",
+		lastName: "",
+		email: "",
+		password: ""
+	};
+
+	// Incorrect email and password validation
+	if (err.message === "Invalid information")
+		errors.email = "Invalid information";
+
+	// In the event we get a duplicate email
+	if (err.code === 11000) {
+		errors.email = "Email is already registered";
+		return errors;
+	}
+
+	// Validation errors
+	if (err.message.includes("user validation failed")) {
+		Object.values(err.errors).forEach((error) => {
+			errors[error.properties.path] = error.properties.message;
+		});
+	}
+
+	return errors;
+};
+
+module.exports.register_post = async (req, res) => {
```

Comment:
```suggestion
module.exports.registerPost = async (req, res) => {
```

to stay consistent on the JS side with camelCase